volmdlr/faces.py

```diff
@@ -2919,14 +3050,15 @@ def contour2d_parametric_to_dimension(self, contour2d: volmdlr.wires.Contour2D,
 
         return volmdlr.wires.Contour2D(primitives2d_dim)
 
-    def contour3d_to_2d_with_dimension(self, contour3d: volmdlr.wires.Contour3D, points_x, points_y):
+    def contour3d_to_2d_with_dimension(self, contour3d: volmdlr.wires.Contour3D,
```

Comment:
for each method for different volmdlr elements, would it be appropriate to move each corresponding method to its object class? we can talk about it if you want

Reply:
I dont think it is a good idea, because it depends on the Surface3D type (_BSpline, Cylindrical_, etc.). We should keep them in the appropriate Surface3D class !